### PR TITLE
Initial Base Class Evaluator

### PR DESCRIPTION
**Pull request by 1Maxnet1**
This PR introduces a base class for a evaluator. If you miss any attributes or functions it should offer, let me know.

### CODE REVIEW

**Review by EhrenbruderRoschtbraten — APPROVED** [review]

File: src/model/evaluator.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+from abc import ABC
+
+
+class Evaluator(ABC):
+    """
+    Base class for an Evaluator
+    """
+
+    def evaluate(self, input_file_path):
```
Comment (LeviUniAcc) [inline src/model/evaluator.py:13]: if it evaluatos on a specific detector we should tell the method which he should use no?
```suggestion
    def evaluate(self, input_file_path: str, detector: str):
```

Comment (LeviUniAcc) [inline src/model/evaluator.py:13]:
```suggestion
    def evaluate(self, input_file_path: str):
```

**Review by RaphaW — APPROVED** [review]
I don't like empty files and would rather prevent them if possible, but other than that it looks good to me 
Thanks

File: src/model/evaluator.py (hunk 1 above)
Comment (1Maxnet1) [reply src/model/evaluator.py:13]: Please see the comment here: https://github.com/Ctrl-Alt-Defeat-Foundation-Models/informatiCup2024/pull/14#discussion_r1385260273

**Comment by 1Maxnet1** [comment]
> I don't like empty files and would rather prevent them if possible, but other than that it looks good to me Thanks

What do you mean with "you don't like empty files" ? The __init__.py file is empty on puprpose see e.g. here: https://docs.python.org/3/tutorial/modules.html#packages

**Review by LeviUniAcc — APPROVED** [review]

**Pull request merged**